client/src/components/DataSpotlight.jsx

```diff
@@ -10,18 +12,84 @@ class DataSpotlight extends React.Component {
     this.percentage = Math.floor(100 * this.buy / this.total);
   }
 
+  // eslint-disable-next-line class-methods-use-this
+  imgError(image) {
+    image.onerror = '';
+    image.src = '../../dist/lib/tag.png';
+  }
+
   render() {
     return (
-      <div>
-        <div className="OjNSjSbJmDliL-he4BWbjSpotlightBG">
-          <br />
-          <h2>tag {this.percentage}%</h2>
-          <p>of {this.total} ratings</p>
-
-        </div>
-      </div>
+      <Circle>
+        <TagPercent>
+          <FlippedTag />
+          &nbsp;
+          {this.percentage}
+          %
+        </TagPercent>
+        <OfNRatings>
+          of {this.total} ratings
+        </OfNRatings>
+      </Circle>
     );
   }
 }
 
+const Circle = styled.div`
+  width: 134px;
+  height: 134px;
+  border-radius: 50%;
+  background-color: rgba(33, 206, 153, 0.30);
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  align-items: center;
+  
+`;
+
+const TagPercent = styled.div`
+  font-family: "DIN Pro", -apple-system, system-ui, sans-serif;
+  font-weight: normal;
+  -webkit-text-stroke: 1px;
+  font-size: 26px;
+  line-height: 30px;
+  text-align: center;
+  letter-spacing: -0.14px;
+  color: #21ce99;
+  display: flex;
+`;
+
+const FlippedTag = styled(Tag)`
+  width: 16px;
+  height: 16px;
+  display: flex;
+  align-self: center;
+  color: #21ce99;
+  
+  -moz-transform: scaleX(-1);
+  -o-transform: scaleX(-1);
+  -webkit-transform: scaleX(-1);
+  transform: scaleX(-1);
+  filter: FlipH;
+  -ms-filter: "FlipH";
+  display: flex;
+`;
+
+const OfNRatings = styled.div`
+  font-weight: normal;
+  font-size: 13px;
+  -webkit-text-stroke: 0.8px;
+  letter-spacing: 0.40px;
+  display: flex;
+  color: #21ce99;
+`;
+
 export default DataSpotlight;
+
+
+{/* 
```

Comment:
looks like this block of code is no longer needed, either add a comment if this will be necessary down the line or just remove it.